release/preview/nanoserver1809/meta.json

```
@@ -5,23 +5,15 @@
     "shortTags": [
         {
             "Tag": "1809"
-        },
-        {
-            "Tag": "1903"
-        },
-        {
-            "Tag": "1909"
-        },
-        {
-            "Tag": "2004"
         }
     ],
     "tagTemplates": [
-        "#psversion#-nanoserver-#tag#",
+        "#psversion#-nanoserver1809-#tag#",
         "preview-nanoserver-#shorttag#"
     ],
     "Base64EncodePackageUrl": true,
     "TestProperties": {
         "size": 1
-    }
+    },
+    "UseAcr": true
```

```suggestion
    }
```

1809 should not use ACR as it can run in normal CI

---

removed